mobile/v1/redshift/sql-runner/sql/standard/03-optional-modules/01-app-errors/01-main/02-app-errors-context.sql

```diff
@@ -0,0 +1,61 @@
+/*
+   Copyright 2021 Snowplow Analytics Ltd. All rights reserved.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+{{if eq (or .enabled false) true}}
+
+  DROP TABLE IF EXISTS {{.scratch_schema}}.mobile_app_errors_context{{.entropy}};
```

Comment:
I believe it's an event, not a context, so should be named differently- I might be mistaken?

Reply:
Good spot. Will rename accordingly